### PR TITLE
ARCHER-11 Added default template layout and a form to add teams

### PR DESCRIPTION
**Pull request by darieldejesus**

### CODE REVIEW

File: src/Template/Layout/default.ctp [hunk 1]
```diff
@@ -1,57 +1,391 @@
 <?php
```
Comment (wacevedo) [inline src/Template/Layout/default.ctp]: im not sure about call this Layout "default.ctp", the name is not very descriptive

File: src/Template/Layout/default.ctp [hunk 2]
```diff
@@ -1,57 +1,391 @@
...
+          <input type="text" name="q" class="form-control" placeholder="Search...">
+              <span class="input-group-btn">
+                <button type="submit" name="search" id="search-btn" class="btn btn-flat"><i class="fa fa-search"></i>
+                </button>
+              </span>
+        </div>
+      </form>
+      <!-- /.search form -->
+
+      <!-- Sidebar Menu -->
```
Comment (wacevedo) [inline src/Template/Layout/default.ctp]: Can we split this sidebar menu on this branch? or is better do it on another?

Comment (darieldejesus) [inline src/Template/Layout/default.ctp]: @wacevedo Would be better if we create another ticket/issue to do this. :)

**Comment by csuriel** [comment]
@wacevedo Please create another ticket to handle it.
Closing PR.

**Comment by darieldejesus** [comment]
@wacevedo @csuriel Why did you close this PR? The requirement Wilson mentioned is about sidebar menu, this PR is for Team form (functionality to add a team).

**Comment by csuriel** [comment]
@darieldejesus looking at the code, was a misunderstanding.
@wacevedo could you create another ticket to separate templates for header and sidebar so that we can use them on all views?

**Comment by darieldejesus** [comment]
Thanks

**Pull request merged**